lib/ansible/modules/copy.py

```diff
@@ -82,6 +82,7 @@
     - When doing a recursive copy set the mode for the directories.
     - If this is not set we will use the system defaults.
     - The mode is only set on directories which are newly created, and will not affect those that already existed.
+    - parameter requires a numeric permissions mode in octal format.
```

Comment:
The value should be quoted to work properly.

Reply:
Thank you for your attention and changes